src/domain/lottoNumbers.js

```diff
@@ -0,0 +1,62 @@
+import {
+  LINE_SIZE,
+  LINE_MINIMUM_NUMBER,
+  LINE_MAXIMUM_NUMBER,
+} from '../constants/purchase.js';
+
+const hasDuplicatedNumbers = (winningNumbersArray) => {
+  return new Set(winningNumbersArray).size !== winningNumbersArray.length;
+};
+
+const isInNumberRange = (numbers) => {
+  const outOfRangeNumbers = numbers.filter(
+    (number) => number < LINE_MINIMUM_NUMBER || number > LINE_MAXIMUM_NUMBER,
+  );
+
+  return outOfRangeNumbers.length > 0;
+};
+
+const isInWinningNumbers = (bonusNumber, winningNumbers) => {
+  const winningNumbersArray = winningNumbers
+    .split(',')
+    .map((number) => Number(number));
+  return winningNumbersArray.includes(bonusNumber);
+};
+
+export const isValidWinningNumbers = (winningNumbers) => {
+  const winningNumbersArray = winningNumbers.split(',');
+
+  const numbersWithoutEmpty = winningNumbersArray.filter(
+    (number) => number !== '',
+  );
```

Comment:
소소하게 pipeline으로 한번에 처리해도 좋을거 같네용 ㅎㅎ

```suggestion
  const winningNumbersArray = winningNumbers
                                                .split(',')
                                                .filter(Boolean)
  );
```

Reply:
아 짚어주셔서 감사합니다! winningNumbersArray 를 재사용하지 않으니 이렇게 써도 좋겠네요.

리뷰어님께서는 체이닝을 5개 이상 길게 쓰시기도 하시는지 궁금합니다.

[8bed9e6](https://github.com/next-step/js-lotto/pull/324/commits/8bed9e6990bd0ba5b99f5328b279a26aa0d8554e)

Reply:
필요한 경우라면 체이닝을 사용할거 같습니다! 5개 이상 길게 쓰는 경우까지는 잘 생각나지 않지만요 ㅎㅎ